plans/install.pp

```diff
@@ -26,7 +26,7 @@
 
   # Common Configuration
   String                            $console_password,
-  String                            $version                          = '2019.8.8',
+  Peadm::Pe_version                 $version                          = '2019.8.8',
```

Comment:
This should probably be `2019.8.10` or `2021.5.0`. Speaking of which, can we make the default auto-updating somehow?

Reply:
We should ideally make auto-updating a thing, yeah. That's probably a piece of work by itself, and would ideally include/encompass plan parameter version defaults, CI job version defaults, and validation functions.